app/scripts/views/tooltip.js

```diff
@@ -42,10 +44,16 @@ define([
       displayedTooltip = this;
 
       var tooltipContainer = this.invalidEl.closest('.input-row,.select-row-wrapper');
+
+      this.$el.addClass(this.extraClassNames);
       this.$el.appendTo(tooltipContainer);
 
       this.setPosition();
 
+      if (this.dismissible) {
+        this.$el.append('<span class="dismiss">&#10005;</span>');
```

Comment:
This adds the [ X ] to the tooltip for the purposes of adding a "dismiss" metrics event. However the tooltip may be dismissed in other ways, such as some other tooltip might appear. @ryanfeeley should we worry about that?